File: code/Combination_of_A_and_B.java
import java.util.ArrayList;
import java.util.Arrays;
import java.util.List;

public class Combination_of_A_and_B {
    public static void main(String[] args) {
        List<String> A = new ArrayList<>(Arrays.asList("a", "b", "c", "d", "e"));
        List<String> B = new ArrayList<>(Arrays.asList("f", "e", "h", "b", "e"));

        System.out.println("Liste A: " + A);
        System.out.println("Liste B: " + B);

        A.addAll(B); // Inserts the elements of list B at the end of list A.
        System.out.println("Combination of A and B: " + A);
    }
}
// Ausgabe:
// Liste A: [a, b, c, d, e]
// Liste B: [f, e, h, b, e]
// Combination of A and B: [a, b, c, d, e, f, e, h, b, e]

File: code/Combination_of_A_and_B.kt
fun main() {
    val a = mutableListOf("a", "b", "c", "d", "e")
    val b = mutableListOf("f", "e", "h", "b", "e")

    println("Liste a: $a")
    println("Liste b: $b")

    a.addAll(b) // Inserts the elements of list b at the end of list a.
    println("Combination of a and b: $a")
}
// Ausgabe:
// Liste a: [a, b, c, d, e]
// Liste b: [f, e, h, b, e]
// Combination of a and b: [a, b, c, d, e, f, e, h, b, e]